src/Parse/ParseQuery.php

```diff
@@ -623,6 +623,26 @@ public function withinGeoBox($key, $southwest, $northeast)
     }
 
     /**
+     * Add a constraint to the query that requires a particular key's
+     * coordinates be contained within a given polygon
+     *
+     * @param string        $key       The key of the ParseGeoPoint
+     * @param array         $points    Array of ParseGeoPoint
+     *
+     * @return ParseQuery Returns this query, so you can chain this call.
+     */
+    public function withinPolygon($key, $points)
+    {
+        $this->addCondition(
+            $key,
+            '$geoWithin',
+            ['$polygon' => $points]
+        );
+
+        return $this;
+    }
+
```

Comment:
I know and postgres has limited support that isn't documented.
The parse-server test's suite had it_exclude_dbs(['postgres'])('byte work') to prevent failing on postgres. Other than that php looks good.